BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -667,12 +667,11 @@ - (void)performRequestQueue:(NSMutableDictionary *)parameters canBatchThisEvent:
     if([self validateSDKTrackingRequirements] == false) {
         return;
     }
-    
-    parameters[kBrowserPlatform] = [BlueShiftDeviceData currentDeviceData].operatingSystem;
-    
-    if (parameters != nil) {
+    NSMutableDictionary* mutableParams = [parameters mutableCopy];
+    [mutableParams setValue:[BlueShiftDeviceData currentDeviceData].operatingSystem forKey:kBrowserPlatform];
+    if (mutableParams != nil) {
```

Comment:
do we need this check here? you are already accessing this value in the above line

Reply:
Thanks for pointing this out. This `if` check is meant for parameters. I will make this change in the code.